app/api/tests/test_dish_api.py

```diff
@@ -101,13 +101,11 @@ def test_creation_of_dish_object(self):
     def test_retrieval_of_one_dish_object(self):
         # Retrieve with valid id
         expected = {
-            'dish': {
-                'name': self.data['name']
-            }
+            'name': self.data['name']
```

Comment:
Please, is there a specific need for changing the format of `expected`. I think it's better we stay consistent to this format as there are several occurrences of this format in other test cases.